qiskit_sphinx_theme/__init__.py

```diff
@@ -13,9 +13,16 @@
 """Pytorch Sphinx theme."""
 
```

Comment:
Ah, you also need to add `from __future__ import annotations` to the top of the file, like the other files. That tells Python to not try to evaluate the type hints at runtime. See https://peps.python.org/pep-0563/ about why it exists.